examples/bridge/src/application/actor.rs

```diff
@@ -46,10 +48,9 @@ impl<R: Rng + Spawner, H: Hasher, Si: Sink, St: Stream> Application<R, H, Si, St
             Self {
                 context,
                 indexer: config.indexer,
-                prover: config.prover,
-                other_prover: config.other_prover,
-                public: poly::public(&config.identity).serialize(),
-                other_public: config.other_network.serialize(),
+                namespace: config.namespace,
+                public: *poly::public(&config.identity),
+                other_public: config.other_identity,
```

Comment:
I guess the types get unified here but seems unnecessary for them to be different in the config in the first place

Reply:
Yeah, I fixed already 👍